.jazzy.yaml

```diff
@@ -10,15 +10,3 @@ module: VelocidiSDK
 output: ./docs/docs_page/
 theme: docs/velocidi-jazzy-theme/
 readme: ./docs/tmp/readme/README.md
-documentation: 
-  - ./docs/*.md 
-  - ./docs/tmp/docs/*.md 
-custom_categories: 
```

Comment:
Do we still need the build steps that split the Readme?

Reply:
Ok, it seems we would still require this script for the `README` part that is used as the main page of the API documentation, and the rest of the partial files are used elsewhere on our documentation building pipelines. So, this is ok.